public/build.css

```diff
@@ -1 +1,4 @@
start}.lg\:items-end{align-items:flex-end}.lg\:items-center{align-items:center}.lg\:items-stretch{align-items:stretch}.lg\:justify-start{justify-content:flex-start}.lg\:justify-end{justify-content:flex-end}.lg\:justify-between{justify-content:space-between}.lg\:gap-4{gap:1rem}.lg\:gap-8{gap:2rem}.lg\:gap-12{gap:3rem}.lg\:gap-32{gap:8rem}.lg\:gap-16{gap:4rem}.lg\:gap-y-16{row-gap:4rem}.lg\:gap-x-16{-moz-column-gap:4rem;column-gap:4rem}.lg\:space-x-4>:not([hidden])~:not([hidden]){--tw-space-x-reverse:0;margin-right:calc(1rem*var(--tw-space-x-reverse));margin-left:calc(1rem*(1 - var(--tw-space-x-reverse)))}.lg\:space-x-10>:not([hidden])~:not([hidden]){--tw-space-x-reverse:0;margin-right:calc(2.5rem*var(--tw-space-x-reverse));margin-left:calc(2.5rem*(1 - var(--tw-space-x-reverse)))}.lg\:border-t-0{border-top-width:0}.lg\:border-b-0{border-bottom-width:0}.lg\:border-l{border-left-width:1px}.lg\:bg-gray-100{--tw-bg-opacity:1;background-color:rgb(243 244 246/var(--tw-bg-opacity))}.lg\:p-4{padding:1rem}.lg\:p-12{padding:3rem}.lg\:p-24{padding:6rem}.lg\:py-32{padding-top:8rem;padding-bottom:8rem}.lg\:px-8{padding-left:2rem;padding-right:2rem}.lg\:py-12{padding-top:3rem;padding-bottom:3rem}.lg\:px-16{padding-left:4rem;padding-right:4rem}.lg\:py-24{padding-top:6rem;padding-bottom:6rem}.lg\:py-16{padding-top:4rem;padding-bottom:4rem}.lg\:pt-8{padding-top:2rem}.lg\:pt-24{padding-top:6rem}.lg\:pl-12{padding-left:3rem}.lg\:pr-12{padding-right:3rem}.lg\:pb-0{padding-bottom:0}.lg\:pl-8{padding-left:2rem}.lg\:pr-0{padding-right:0}.lg\:pr-8{padding-right:2rem}.lg\:text-left{text-align:left}.lg\:text-right{text-align:right}.lg\:text-3xl{font-size:1.875rem;line-height:2.25rem}.lg\:text-lg{font-size:1.125rem;line-height:1.75rem}.lg\:text-xs{font-size:.75rem;line-height:1rem}.lg\:font-bold{font-weight:700}.lg\:uppercase{text-transform:uppercase}.lg\:tracking-wide{letter-spacing:.025em}.lg\:text-gray-500{--tw-text-opacity:1;color:rgb(107 114 128/var(--tw-text-opacity))}.lg\:opacity-0{opacity:0}.lg\:transition{transition-property:color,background-color,border-color,fill,stroke,opacity,box-shadow,transform,filter,-webkit-text-decoration-color,-webkit-backdrop-filter;transition-property:color,background-color,border-color,text-decoration-color,fill,stroke,opacity,box-shadow,transform,filter,backdrop-filter;transition-property:color,background-color,border-color,text-decoration-color,fill,stroke,opacity,box-shadow,transform,filter,backdrop-filter,-webkit-text-decoration-color,-webkit-backdrop-filter;transition-timing-function:cubic-bezier(.4,0,.2,1);transition-duration:.15s}.lg\:transition-all{transition-property:all;transition-timing-function:cubic-bezier(.4,0,.2,1);transition-duration:.15s}.group:hover .lg\:group-hover\:absolute{position:absolute}.group:hover .lg\:group-hover\:relative{position:relative}.group:hover .lg\:group-hover\:opacity-0{opacity:0}.group:hover .lg\:group-hover\:opacity-100{opacity:1}}@media (min-width:1280px){.xl\:max-w-\[1348px\]{max-width:1348px}}
```

Comment:
This file is re-compiled on every deployment and therefore any changes would be removed.